developer-docs-site/docs/CODEOWNERS

```diff
@@ -0,0 +1,17 @@
+# This is the overarching CODEOWNERS file for Aptos.dev documentation.
+# It exists to help route review requests and ensure proper review of changes.
+# We include each subdirectory and relevant owners below:
+# Global rule:
+* @clay-aptos
+## Aptos White Paper
+/aptos-white-paper/** @aching @ShaikhAliMo
+## CLI Tools
+/cli-tools/** @gregnazario @davidiw
+## Aptos Concepts
+/concepts/** @movekevin
+## Dev Guides
+/guides/** @movekevin
+## Nodes
+/nodes/** @rustielin @aptos-labs/@prod-eng
+## Reference
+/reference/** @clay-aptos @gregnazario
```

Comment:
add a newline just in case

Reply:
Christian, are you suggesting a new line between each directory entry and preceding comment? Thanks for your review!

Reply:
This is fine for now but we should make an Ecosystem Platform group and use that in place of Greg specifically, e.g. for `/cli-tools/`.

Reply:
> This is fine for now but we should make an Ecosystem Platform group and use that in place of Greg specifically, e.g. for `/cli-tools/`.

Good point, Daniel. @gregnazario , can we create such a group? Do we need more? Thanks!

Reply:
> add a newline just in case

no, I just meant a newline at the end of the file to get rid of this:
<img width="476" alt="image" src="https://user-images.githubusercontent.com/1221897/202794649-1e76abb7-0ed0-49f1-b79b-7396e4738c80.png">

Reply:
> > Christian, are you suggesting a new line between each directory entry and preceding comment?
> 
> no, I just meant a newline at the end of the file to get rid of this: <img alt="image" width="476" src="https://user-images.githubusercontent.com/1221897/202794649-1e76abb7-0ed0-49f1-b79b-7396e4738c80.png">

Done. Thanks!